test/test_data/firefox.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2022 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+mapping_test_data = [
+    {
+        "product": "firefox",
+        "version": "106.0",
+        "version_strings": [r"ENCODER=Mozilla106.0"],
+    },
+]
+package_test_data = [
+    {
+        "url": "https://www.rpmfind.net/linux/rpm2html/search.php?query=firefox",
```

Comment:
```suggestion
        "url": "https://www.rpmfind.net/linux/fedora/linux/development/rawhide/Everything/aarch64/os/Packages/f/",
```

Here the url + package_name should be a valid link that is pointed to the package. Then only will the tool be able to fetch the package and create a condensed file.

Reply:
> I just noticed that you've renamed the filename instead of adding a different condensed file. That will also lead to test failure.
> 
> Here's what you should do
> 
> - Delete the rpm condensed file.
> - Run long tests and see if all tests are passing.
> - Commit the condensed file along with any changes.

This is exactly what I did. I did not rename the file.